libtransact/src/state/merkle/sql/operations/get_path.rs

```diff
@@ -0,0 +1,290 @@
+/*
```

Comment:
Typo in commit message "Add MerkRadix..." should be "Add MerkleRadix..."

Reply:
Fixed